### PR TITLE
Gabor review fixes 5

### PR DESCRIPTION
**Pull request by GBuella**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmemfile/137)

### CODE REVIEW

**Comment by krzycz** [comment]
:lgtm:

---

Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/137#-:-KkLVtb_mcURNlBw4lYH:bnfp4nl)*

File: src/libpmemfile/path_resolve.c [hunk 1]
```diff
@@ -110,7 +110,7 @@ resolve_symlink(struct resolved_path *result,
 		}
 	}
 
-	if (! *is_last_component)
+	if (!(*is_last_component))
```
Comment (marcinslusarz) [inline src/libpmemfile/path_resolve.c]: I don't see any point in doing that. If anything can be improved here it's space removal.

Comment (GBuella) [inline src/libpmemfile/path_resolve.c]: In that case, please responde to Sarah's comment about it in:
https://github.com/pmem/pmemfile/pull/95

**Comment by sarahjelinek** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/137#-:-KkMhIoeLozRyY65ywO1:bnfp4nl)*

**Comment by GBuella** [comment]
I have no idea how the "travis: update syscall_intercept reference" got here.

**Pull request merged**